tests/Spout/Writer/ODS/WriterWithStyleTest.php

```diff
@@ -237,6 +240,62 @@ public function testAddRowWithStyleShouldApplyWrapTextIfCellContainsNewLine()
     }
 
     /**
+     * @return void
+     */
+    public function testBorders()
+    {
+        $fileName = 'test_borders.ods';
+
+        $dataRows = [
+            ['row-with-border-bottom-green-thick-solid'],
+            ['row-without-border'],
+            ['row-with-border-top-red-thin-dashed'],
+        ];
+
+        $borderBottomGreenThickSolid = (new BorderBuilder())
+            ->setBorderBottom(Border::STYLE_SOLID, Color::GREEN, Border::WIDTH_THICK)->build();
+
+
+        $borderTopRedThinDashed = (new BorderBuilder())
+            ->setBorderTop(Border::STYLE_DASHED, Color::RED, Border::WIDTH_THIN)->build();
+
+        $styles =  [
+            (new StyleBuilder())->setBorder($borderBottomGreenThickSolid)->build(),
+            (new StyleBuilder())->build(),
+            (new StyleBuilder())->setBorder($borderTopRedThinDashed)->build(),
+        ];
+
+        $this->writeToODSFileWithMultipleStyles($dataRows, $fileName, $styles);
+
+        $styleElements = $this->getCellStyleElementsFromContentXmlFile($fileName);
+        $this->assertEquals(3, count($styleElements), 'There should be 3 styles)');
+
+        $expectedFirst = sprintf(
+            '%s %s #%s',
+            BorderHelper::$widthMap[Border::WIDTH_THICK],
+            BorderHelper::$styleMap[Border::STYLE_SOLID],
+            Color::GREEN
+        );
+
+        $this->assertEquals($expectedFirst, $styleElements[1]
```

Comment:
move the actual value retrieval outside of the `assertEquals` as it's a bit hard to read